data/json/regional_map_settings.json

```diff
@@ -544,6 +544,31 @@
           "mx_portal_in": 1,
           "mx_anomaly": 3
         }
+      },
+      "research_facility_lot": {
+        "chance": 3,
+        "extras": {
+          "mx_helicopter": 40,
+          "mx_military": 65,
+          "mx_collegekids": 1,
+          "mx_portal": 160,
+          "mx_crater": 180,
+          "mx_fumarole": 10,
```

Comment:
Is it possible for fumaroles to burn down the whole building, and if so, is that ok?

Reply:
I don't see why not, to both

Reply:
That's fine then.